test/integration/main_test.go

```diff
@@ -70,42 +74,31 @@ func configV1Client() (result *configv1client.ConfigV1Client) {
 	return client
 }
 
-func clusterOperatorInsights() *configv1.ClusterOperator {
-	// get info about insights cluster operator
-	operator, err := configClient.ClusterOperators().Get("insights", metav1.GetOptions{})
+func clusterOperator(clusterName string) *configv1.ClusterOperator {
+	// get info about given cluster operator
+	operator, err := configClient.ClusterOperators().Get(clusterName, metav1.GetOptions{})
 	if err != nil {
+		// TODO -> change to t.Fatal in follow-up PR
```

Comment:
Do we really want a followup PR when backporting? :)

Reply:
I want to keep it as similar to the version above as possible.

Reply:
Yeah ok makes sense.